solutions/9.js

```diff
@@ -0,0 +1,12 @@
+const prime = (a,i=2)=>{
+  if(a%i == 0){
+    if(i==a){
+      return "its a prime number";
```

Comment:
It would be easier to just return true here. Then you wouldn't have to use the whole string in the test.